.buildkite/pipeline.ml.yml

```diff
@@ -318,6 +318,13 @@
     - DATA_PROCESSING_TESTING=1 PYTHON=3.7 ./ci/travis/install-dependencies.sh
     - bazel test --config=ci $(./scripts/bazel_export_options) --build_tests_only python/ray/workflow/... python/ray/data/...
 
+- label: ":slot_machine: ML Utils tests"
+  conditions: ["RAY_CI_ML_UTILS_AFFECTED"]
+  commands:
+    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
+    - TUNE_TESTING=1 ./ci/travis/install-dependencies.sh
```

Comment:
nit: Do we need `TUNE_TESTING` here or want something more granular?

Reply:
This is a bit more involved since we would have to create a `requirements_ml_utils.txt` file and then have to figure out how to keep the `mlflow` versions in sync between train/tune/ml_utils. I think we can leave this for a future PR.